pulsar-broker/src/main/java/org/apache/pulsar/broker/transaction/buffer/impl/TopicTransactionBuffer.java

```diff
@@ -322,8 +322,12 @@ private void takeSnapshot() {
         });
     }
 
-
-
+    private void clearAbortTransaction() {
+        while (!aborts.isEmpty() && !((ManagedLedgerImpl) topic.getManagedLedger())
+                .ledgerExists(aborts.get(aborts.firstKey()).getLedgerId())) {
+            aborts.remove(aborts.firstKey());
```

Comment:
Should be log the txid of the transactions that we are clearing?

Reply:
good, i will add the debug level log for clear aborted transaction.